README.md

```diff
@@ -1338,9 +1338,29 @@ My name is Yuliia and i'm fond of tigers. They're also cats (even big though :gr
 
 ![Tiger](https://media.giphy.com/media/3gPNRFAf1r1108GsK7/giphy-downsized-large.gif)
 ***
+<<<<<<< HEAD
```

Comment:
This got into the commit due to an incorrectly fixed conflict, you need to remove it. And clean up the code below.